### PR TITLE
by using pipenv 

### PR DESCRIPTION
**Pull request by teitei-tk**
# Description
* By using pipenv, package management can be done with python like Gemfile in ruby.
  * https://github.com/pypa/pipenv

## documents
* https://docs.pipenv.org/basics/

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+"boto3" = "*"
+onelogin = "*"
+keyring = "*"
+
+[requires]
```
Comment (slycoder) [inline Pipfile]: Should specify that we need python 3

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+name = "pypi"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+"boto3" = "*"
```
Comment (slycoder) [inline Pipfile]: Is there any reason not to use the same constraints as requirements.txt?

File: Pipfile (hunk 1 above)
Comment (teitei-tk) [reply Pipfile]: Is python version 3.5 or later?.  i think so from travis.yml file.

https://github.com/physera/onelogin-aws-cli/blob/101c30c7dd31f7e84554007c0efc022c1c81c3fe/.travis.yml#L2

File: Pipfile (hunk 2 above)
Comment (teitei-tk) [reply Pipfile]: I do not understand. I generated lockfile based on requirements.txt, but somehow I have double quotation.

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+onelogin = "*"
+keyring = "*"
+
+[requires]
+python_version = ">=3.5"
```
Comment (teitei-tk) [inline Pipfile]: goodbye python2, hello python3 !

File: Pipfile (hunk 2 above)
Comment (slycoder) [reply Pipfile]: Oh, I meant the versioning.  But now looking at the docs it seems like pipenv's recommendation is to unpin things here are rely on the lockfile.

**Pull request merged**